PythonRegression/util/static_vals.py

```diff
@@ -4,28 +4,59 @@
 
 TEST_NEIGHBORS = [u'udp://178.128.236.6:14600',u'udp://167.99.178.3:14600'] 
 
-TEST_HASH = ["KEHXTIVIBJHIQOWXQ9BSDMIDXLNO9WJQUGOSKVTMFMAHNSAYDSFWGOYQWINLCAMBASPUATDNVCFT99999"]
+TEST_HASH = "NMPXODIWSCTMWRTQ9AI9VROYNFACWCQDXDSJLNC9HKCECBCGQTBUBKVROXZLQWAZRQUGIJTLTMAMSH999"
+
+TEST_TIP_LIST = ["SBKWTQWCFTF9DBZHJKQJKU9LXMZD9BMWJIJLZCCZYJFWIBGYYQBJOWWFWIHDEDTIHUB9PMOWZVCPKV999"]
 
 TEST_ADDRESS = "TEST9TRANSACTION9TEST9TRANSACTION9TEST9TRANSACTION9TEST9TRANSACTION9TEST999999999"
+TEST_EMPTY_ADDRESS = "EMPTY9BALANCE9TEST999999999999999999999999999999999999999999999999999999999999999"
+TEST_STORE_ADDRESS = "STORE9AND9FIND9999999999999999999999999999999999999999999999999999999999999999999"
 
```

Comment:
A cool thing I found:
http://code.activestate.com/recipes/65207-constants-in-python/?in=user-97991

However, the zen of python implies an attitude that one should know what he is doing and don't need extra protections, so you don't have to use the solution in the link (just wanted to share).

Anyhows even when you declare constants in step files make them upper case to make sure no one changes them.